ArduCopter/mode_auto.cpp

```diff
@@ -1512,6 +1512,16 @@ bool ModeAuto::verify_land()
         case State::Descending:
             // rely on THROTTLE_LAND mode to correctly update landing status
             retval = copter.ap.land_complete && (motors->get_spool_state() == AP_Motors::SpoolState::GROUND_IDLE);
+            if (retval && !mission.continue_after_land() && copter.motors->armed()) {
```

Comment:
I'm not sure this is correct because it won't properly spool down the motors.  I think that instead we need to add something into ModeAuto::land_run() and ModeAuto::rtl_run().

For ModeAuto::rtl_run() I think we should send true or false to copter.mode_rtl.run() depending upon the value of the mission option.

Reply:
this code only runs if (motors->get_spool_state() == AP_Motors::SpoolState::GROUND_IDLE) is true, so doesn't that mean the motors are spooled down before this check is true?

Reply:
sorry, this looks fine actually as discussed on the dev call.